draft-api/src/test/scala/no/ndla/draftapi/service/WriteServiceTest.scala

```diff
@@ -1559,4 +1559,33 @@ class WriteServiceTest extends UnitSuite with TestEnvironment {
     result.status.current should be(IN_PROGRESS.toString)
     result.started should be(true)
   }
+
+  test("That partial published fields does set started") {
```

Comment:
Antar det skulle stått not her?
```suggestion
  test("That partial published fields does not set started") {
```